### DISCUSSION TITLE
About my issue in Reanimated

### DISCUSSION THREAD

**Discussion by enesozturk** [enesozturk/react-native-hold-menu#11] [General]
Hi, in recent weeks I got an issue in Reanimated when I try to render menu list. When I press MenuItem, it calls `onPress` of object in the list. It gives error. Same think then I try to render `icon` prop. 

@ksitko, I wonder which Reanimated version you used when run the examples? The error is happening after **>v2.0.0-rc.0**.

Also @gorhom have you ever got that issue before or do you have an idea?

Issue: https://github.com/software-mansion/react-native-reanimated/issues/1758

**Comment by ksitko** [comment]
@enesozturk Sorry, I cloned the repo and used the exact version in that it provided. I haven't yet gotten the chance to integrating the library into my current app.

**Reply by enesozturk** [reply]
I see, I thought maybe you tried in another app. It is important problem for **>=v2.0.0-rc.1**. Waiting response from the team.

**Reply by ksitko** [reply]
Glad to it got resolved and will be in the next Reanimated V2 release!

**Reply by enesozturk** [reply]
Yess, that is awesome 🔥

**Reply by ksitko** [reply]
Have you tested 2.0.1? I'm getting the same error.

**Reply by enesozturk** [reply]
Oh really? Have not checked yet. Will check soon.

**Reply by enesozturk** [reply]
Unfortunately yes 👎🏽 Cannot see this improvement in release notes, maybe they did not published it yet? I will check it.

**Reply by ksitko** [reply]
I thought they merged the improvement days before the release, was it not included? Strange.
